molecule/default/tests/test_default.py

```diff
@@ -17,12 +16,15 @@
     "f", ["/usr/bin/python", "/usr/bin/python2", "/usr/bin/python2.7"]
 )
 def test_python(host, f):
-    """Ensure that python2-specific files no longer exist, except on AmazonLinux or Debian 9."""
+    """Ensure that python2-specific files no longer exist, except on AmazonLinux or Debian 9/10."""
```

Comment:
I think this doc string should specifically mention "stretch" and "buster", not just "Debian 9/10", since we are no longer matching by Debian release number.